Gruntfile.js

```diff
@@ -49,7 +49,7 @@ module.exports = function(grunt) {
       options: {
         accessKeyId: '<%= aws.key %>',
         secretAccessKey: '<%= aws.secret %>',
-        bucket: 'demos.algorithmia.com',
+        bucket: 'algorithmia-synapse',
```

Comment:
🚨 This seems like it would upload files to the wrong bucket?

Reply:
This is intentional. I'm testing that everything works in the Synapse bucket before updating to the real bucket. Wanted to make sure there aren't any 404s before I publish for real.

Reply:
Ok roger

Reply:
Ah just read through your full PR description. Come on Ryan, read the instructions

Reply:
Last thought: going forward, let's definitely create fresh test buckets for these use-cases. With that Synapse bucket now serving Hubspot I'd like to treat it as a production resource.

Reply:
The `algorithmia-synapse` bucket isn't used for anything, so it's safe to tinker around with. When I wired up the Synapse CDN I found out after creating that bucket that the bucket name has to match the custom domain in order for everything to work. The bucket we're using for HubSpot is `cdn.algorithmia.design`.

Once this is PR is merged I'll delete this unused bucket to prevent any future confusion. 100% agree on using fresh buckets moving forward.

Reply:
Ah, thank you for clarifying! Glad that it was safer than I anticipated.